rl.py

```diff
@@ -2,13 +2,26 @@
 This is a file that will run a simple rl model.
 """
 
+import expe
+import models
+
 def main():
     '''
-    This function will just print hello world for now.
+    This function will define an expe, run models on it, analyze it.
```

Comment:
Don't need to start docstrings with "this function will". Just say "Define an experiment, run...".